dumper/app/build.gradle

```diff
@@ -69,16 +72,13 @@ dependencies {
     implementation libs.aws.java.sdk.redshift
     implementation libs.aws.java.sdk.cloudwatch
 
-    runtimeOnly libs.logback.classic
+    implementation libs.logback.classic
```

Comment:
This should remain runtimeOnly - it's the runtime implementation, not the API. The API is all in the slf4j packages.

If you need this in the implementation, something else is wrong. I assume you're familiar with the images in https://docs.gradle.org/current/userguide/java_library_plugin.html ?

Reply:
Followup notes: If it's not obvious, what caused you to put this in the implementation? Did you get a LinkageError of some sort? Did someone statically link against a logback class? Happy to help, just please let me know what triggered this change, please.

Reply:
changed back to `runtimeOnly`